src/lib/abacus/dydxChainTransactions.ts

```diff
@@ -106,13 +130,138 @@ class DydxChainTransactions implements AbacusDYDXChainTransactionsProtocol {
     throw new Error(`Unsupported data type: ${typeof x}`);
   }
 
+  async placeOrderTransaction(params: HumanReadablePlaceOrderPayload): Promise<string> {
+    if (!this.compositeClient || !this.localWallet)
+      throw new Error('Missing compositeClient or localWallet');
+
+    try {
+      const {
+        subaccountNumber,
+        marketId,
+        type,
+        side,
+        price,
+        size,
+        clientId,
+        timeInForce,
+        goodTilTimeInSeconds,
+        execution,
+        postOnly,
+        reduceOnly,
+        triggerPrice,
+      } = params || {};
+
+      // Observe uncommitted order
+      this.store?.dispatch(addUncommittedOrderClientId(clientId));
+
+      setTimeout(() => {
+        this.store?.dispatch(removeUncommittedOrderClientId(clientId));
```

Comment:
I am a bit confused, it seems to me that `removeUncommittedOrderClientId` is only being called on timeout/error. Is uncommittedOrderClientId still being used?

Reply:
oh nvm, I see it being unset in `setLatestOrder`